### ISSUE TITLE
SystemUiOverlayStyle#systemNavigationBarIconBrightness doesn't work

### ISSUE THREAD

**Issue by onlymash** [flutter/flutter#29999]
## code 
```
class App extends StatelessWidget {
  final SystemUiOverlayStyle _currentStyle =  SystemUiOverlayStyle.light.copyWith(
    statusBarColor: Colors.grey[50],
    statusBarBrightness: Brightness.light,
    systemNavigationBarColor: Colors.grey[50],
    systemNavigationBarIconBrightness: Brightness.light,
  );
  @override
  Widget build(BuildContext context) {
    SystemChrome.setSystemUIOverlayStyle(_currentStyle);
    return MaterialApp(
      title: 'Flexbooru',
      theme: ThemeData(
        primaryColor: Colors.deepPurple,
        primaryColorDark: Colors.grey[50],
        primaryColorBrightness: Brightness.light,
      ),
      home: Home(),
    );
  }
}
```

## screenshot
Flutter version:
[Screenshot_20190327-054627_Flexbooru](https://user-images.githubusercontent.com/5838634/55035910-1584a400-5054-11e9-9cd2-b05cd76b58ad.png)

Native version:
`<item name="android:windowLightNavigationBar">true</item>`

[Screenshot_20190327-054640_Flexbooru](https://user-images.githubusercontent.com/5838634/55035941-2503ed00-5054-11e9-8d66-a1e645e495a5.png)

**Comment by HansMuller** [comment]
CC @jonahwilliams

**Comment by VladyslavBondarenko** [comment]
Tried with current master `1.21.0-6.0.pre.41` on Android 9 device.
It's working as intended.
The navigation buttons are light when I set `systemNavigationBarIconBrightness: Brightness.light` and they are dark when I set `systemNavigationBarIconBrightness: Brightness.dark`.

Possibly needs to be mentioned in https://api.flutter.dev/flutter/services/SystemUiOverlayStyle/systemNavigationBarIconBrightness.html
